open-hackathon-client/src/client/static/js/views/oh.home.js

```diff
@@ -50,6 +50,65 @@
                 infn3.removeClass('hide');
             }
         })
+
+        oh.api.hackathon.notice.list.get({
+            query: {
+                order_by: 'time',
+                show: 'pagination',
+                page: 1,
+                per_page: 6 //the latest 6 notices are shown
+            }
+        }, function(data) {
+            if(data.error) {
+                alert(data.error.message);
```

Comment:
Please don't use `alert` which is usually blocked by browsers. Try `oh.comm.alert`(name may vary)